app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -259,13 +262,26 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi, RemoveBlo
     showBloodSugarErrorMessage(getString(R.string.bloodsugarentry_error_empty))
   }
 
-  override fun showBloodSugarHighError(measurementType: BloodSugarMeasurementType) {
+  override fun showBloodSugarHighError(measurementType: BloodSugarMeasurementType, unitPreference: BloodSugarUnitPreference) {
+    when (unitPreference) {
+      Mg -> showBloodSugarHighErrorForMg(measurementType)
```

Comment:
We're adding the error for both unit preferences here and the commit message suggests for mmol/L only? Can we change it to add for both the unit preferences?

Reply:
Here we are reusing existing mg error and added a new error for mmol, so that why I didn't mention mg in the commit message.